moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -305,6 +305,13 @@ void MotionPlanningDisplay::reset()
   query_robot_goal_->setVisible(query_goal_state_property_->getBool());
 }
 
+void MotionPlanningDisplay::setName(const QString& name)
+{
+  BoolProperty::setName(name);
```

Comment:
`BoolProperty::setName(name)` sets the Display name.
`setWindowTitle(name)` sets the Panel name or title.
I'm not sure what `setObjectName(name)` does, but the original code from rviz::display says in comments:
"QMainWindow::saveState() needs objectName to be set."